client/src/utils/cache/index.js

```diff
@@ -0,0 +1,113 @@
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import * as FileSystem from 'expo-file-system'
+
+/**
+ * This file contains helper methods related to caching a mapping from vocab items their
+ * audio and image files that are stored in Expo FileSystem. This mapping is stored in AsyncStorage,
+ * and is used to limit the number of GET calls required for fetching image and audio files from the API.
+ */
+
+/* Represents the amount of time in seconds that have to pass before a cached file is invalidated */
+const INVALIDATION_SECONDS = 60 * 60 * 24
+
+/**
+ * Returns the AsyncStorage key encoding given a vocab item and its associated file
+ * @param {String} vocabID Unique MongoDB ID of Vocab Item
+ * @param {String} fileType Image or Audio
+ * @returns
+ */
+const getGetGivenType = (vocabID, fileType) => {
+  switch (fileType) {
+    case 'audio':
+      return `${vocabID}/audio`
+    case 'image':
+      return `${vocabID}/image`
+    default:
+      throw new Error('No file type provided')
+  }
```

Comment:
Curious about the necessity for a whole function that is really just doing `${vocabID}/${fileType}`, not sure if you have any additional thoughts in mind why you are doing this

Reply:
u right